### PR TITLE
feat: Turn on health monitor

### PR DESCRIPTION
**Pull request by litt3**
- cherry pick of #13924

### CODE REVIEW

**Review by mxtartaglia-sl — APPROVED** [review]

**Review by thomas-swirlds-labs — APPROVED** [review]
LGTM. Thanks @litt3

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java [hunk 1]
```diff
@@ -41,11 +41,11 @@
...
 @ConfigData("platform.wiring")
 public record WiringConfig(
-        @ConfigProperty(defaultValue = "false") boolean healthMonitorEnabled,
-        @ConfigProperty(defaultValue = "true") boolean hardBackpressureEnabled,
-        @ConfigProperty(defaultValue = "1.0") double defaultPoolMultiplier,
-        @ConfigProperty(defaultValue = "0") int defaultPoolConstant,
+        @ConfigProperty(defaultValue = "true") boolean healthMonitorEnabled,
+        @ConfigProperty(defaultValue = "false") boolean hardBackpressureEnabled,
+        @ConfigProperty(defaultValue = "0") double defaultPoolMultiplier,
+        @ConfigProperty(defaultValue = "8") int defaultPoolConstant,
```
Comment (OlegMazurov) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java:47]: `PreHandleWorkflowImpl.preHandle()` submits parallel tasks to the current `ForkJoinPool` (i.e. the platform `ForkJoinPool`). Signature verifications are CPU-heavy.  Setting the default parallelism to 8 may be too limiting. 
What's the rationale behind the change?

Comment (litt3) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java:47]: It's what Cody had this set to when he left. The idea was to use these config values as a starting point, and evaluate performance from there.

Comment (OlegMazurov) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java:47]: I suggest not changing them (based on my evaluation of current performance).

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java [hunk 2]
```diff
@@ -36,5 +36,5 @@
 @ConfigData("consensus")
 public record ConsensusConfig(
         @ConfigProperty(defaultValue = "26") int roundsNonAncient,
-        @ConfigProperty(defaultValue = "1500") int roundsExpired,
```
Comment (lpetrovic05) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java:39]: how is this related to the health monitor?

**Comment by lpetrovic05** [comment]
i honestly don't understand this changeset

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java (hunk 2 above)
Comment (litt3) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java:39]: It's not, but I was asked to make the change.

Comment (poulok) [reply platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusConfig.java:39]: I address this with Lazar offline

**Review by poulok — APPROVED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java (hunk 1 above)
Comment (poulok) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WiringConfig.java:47]: Discussed offline, we have agreement to move forward with the current values.
